imap_processing/swe/l2/swe_l2.py

```diff
@@ -136,22 +138,37 @@ def calculate_flux(l1b_dataset: xr.Dataset) -> npt.NDArray:
     """
     Calculate flux.
 
-    To get flux, j = 2 * m * E * f
-        where:
-        f = calculate_phase_space_density() result
-        m - mass of electron
-        E - energy in joules. E(eV) * 1.60219e-19(J/eV)
-
-    To convert flux units:
-        j = 2 * m * E * f
-          = 2 * 9.10938356e-31 kg * E(eV) * 1.60219e-19(J/eV) * (s^3/ (cm^6 * ster)
-          = 2 * 9.10938356e-31 kg * E(eV) * 1.60219e-19 (kg * m^2 / s^2) *
-            (s^3 / (cm^6 * ster)
-          = 2 * 9.10938356e-31 * E(eV) * 1.60219e-19  kg^2 * ( 10^4 cm^2 / s^2) *
-            (s^3 / (cm^6 * ster)
-          = 2 * 9.10938356e-31 * E(eV) * 1.60219e-19 * 10^4 ((kg^2 * cm^2 * s^3) /
-            (s^2 * cm^6 * ster))
-        TODO: ask Ruth Skoug what units to use for flux and work out remaining units.
+    Flux is represented by this symbol, j. Its unit is
+    1 / (2 * eV * cm^2 * s * ster).
+
+    The formula to calculate flux,
+
+    Where:
+        fv = the phase space density of solar wind electrons
+            given by calculate_phase_space_density() result.
+        J = kg * m^2 / s^2. J for joules.
+        E = Energy in Joules. eV * 1.60219e-19(J/eV). eV is result
+            from get_particle_energy() function.
```

Comment:
You don't need this line defining Joules. 

In general, you don't really need to define SI units like Joules, seconds, centimeters, etc., but here it doesn't come into the equation, so you definitely don't need to. 

```suggestion
        eV = Energy in electron-volts, calculated by get_particle_energy().
        E = Energy in Joules. eV * 1.60219e-19(J/eV). 
```